hal/src/boron/cellular_hal.cpp

```diff
@@ -221,6 +223,41 @@ int cellular_credentials_clear(void* reserved) {
     return 0;
 }
 
+cellular_result_t cellular_global_identity(CellularGlobalIdentity* cgi_, void* reserved_)
+{
```

Comment:
Nitpicking: Please fix the formatting to follow the same style as the rest of the file.

Reply:
done.